starknet_devnet/general_config.py

```diff
@@ -15,23 +16,38 @@
 from .constants import SUPPORTED_TX_VERSION
 from .fee_token import FeeToken
 
-DEFAULT_GENERAL_CONFIG = build_general_config(
-    {
-        "cairo_resource_fee_weights": {
-            "n_steps": constants.N_STEPS_FEE_WEIGHT,
-        },
-        "contract_storage_commitment_tree_height": constants.CONTRACT_STATES_COMMITMENT_TREE_HEIGHT,
-        "event_commitment_tree_height": constants.EVENT_COMMITMENT_TREE_HEIGHT,
-        "global_state_commitment_tree_height": constants.CONTRACT_ADDRESS_BITS,
-        "invoke_tx_max_n_steps": DEFAULT_MAX_STEPS,
-        "min_gas_price": DEFAULT_GAS_PRICE,
-        "sequencer_address": hex(DEFAULT_SEQUENCER_ADDRESS),
-        "starknet_os_config": {
-            "chain_id": DEFAULT_CHAIN_ID.name,
-            "fee_token_address": hex(FeeToken.ADDRESS),
-        },
-        "tx_version": SUPPORTED_TX_VERSION,
-        "tx_commitment_tree_height": constants.TRANSACTION_COMMITMENT_TREE_HEIGHT,
-        "validate_max_n_steps": DEFAULT_VALIDATE_MAX_STEPS,
-    }
-)
+
+class StarknetChainId(Enum):
```

Comment:
No need to define this here, it can be imported from `starkware.starknet.definitions.general_config`, the same in `devnet_config.py`

Reply:
true, somehow I missed that

https://github.com/Shard-Labs/starknet-devnet/pull/358/commits/ac47c4805052e3907b5129e156cbe213491b7537